utils/uiTypes/proposalCreationTypes.ts

```diff
@@ -36,17 +36,29 @@ export interface FormInstructionData {
 export interface SplTokenTransferForm {
   destinationAccount: string;
   amount: number | undefined;
-  governedTokenAccount: GovernedTokenAccount | undefined;
+  governedTokenAccount?: GovernedTokenAccount | undefined;
```

Comment:
When it's optional, there is no need for `undefined`? You can go with:

```
export interface SplTokenTransferForm {
  destinationAccount?: string;
  amount?: number;
  governedTokenAccount?: GovernedTokenAccount;
  programId?: string;
  mintInfo?: MintInfo;
}
```

And in SplTokenTransfer:

```
  const [form, setForm] = useState<SplTokenTransferForm>({
    programId: programId?.toString(),
  });
```